src/Altinn.Apps/AppFrontend/react/shared/__tests__/utils/instanceContext.test.ts

```diff
@@ -0,0 +1,48 @@
+import 'jest';
+import { IInstanceContext, IInstance } from '../../src/types';
+import { buildInstanceContext } from '../../src/utils/instanceContext';
+
+describe('>>> src/Altinn.Apps/AppFrontend/react/shared/src/utils/instanceContext.ts', () => {
+  let partyId: string;
+  let appId: string;
+  let instaceId: string;
+
+  let mockInstance: IInstance;
+
+  beforeEach(() => {
+    partyId = '1337';
+    instaceId = `${partyId}/super-secret-uuid-000`;
+    appId = 'tdd/enapp';
+
+    mockInstance = {
+      id: instaceId,
+      appId: appId,
+      instanceOwner: {
+        partyId: partyId,
+      }
+    } as IInstance;
+  });
```

Comment:
Minor comment. The `beforeEach` here is unnecessary as these are never mutated during testing, therefore is no need to reset them after each run. 

Could be refactored to
```suggestion
  const partyId = '1337';
  const appId =  'tdd/enapp';
  const instaceId = `${partyId}/super-secret-uuid-000`;
  const mockInstance: IInstance = {
    id: instaceId,
    appId: appId,
    instanceOwner: {
      partyId: partyId,
    }
  } as IInstance;
```